app/services/idv/proofer.rb

```diff
@@ -13,11 +13,9 @@ module Proofer
 
     STAGES = %i[resolution state_id address].freeze
 
-    class << self
-      attr_accessor :configuration
-
-      @vendors = {}
+    @vendors = {}
```

Comment:
I believe this to be the actual fix here, moving the initialization of @vendors to the appropriate context.